pages/Frequency/Overview.md

```diff
@@ -0,0 +1,38 @@
+# Frequency DSNP Implementation (Pre-Release)
+__Version pre-1.0.0__
+
+_Final release is pending release of [Frequency v1.0.0](https://github.com/LibertyDSNP/frequency)_
+
+DSNP on Frequency is built using Message Source Accounts (MSAs) for identity and delegation with Schemas defined for each Announcement Type.
```

Comment:
I think there is a lot in here; how about:

DSNP on Frequency is built using Message Source Accounts (MSAs) for identity,  delegation to Service Providers, and messages with Schemas defined for each Announcement Type.

Reply:
Nice. Updating.